dominion/frontend/src/component/Footer/SocialMedia.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
Same as Partners. Either

 * Part of Footer: Then Footer should included it and **no other component/page** should know/use it, or
 * Component of its own: Move it to components folder ie. `src/components/SocialMedia.js`.